doc/HttpLuaModule.wiki

```diff
@@ -4672,12 +4689,15 @@ This method was introduced in the <code>0.5.0rc30</code> release.
 
 == ngx.escape_uri ==
 
-'''syntax:''' ''newstr = ngx.escape_uri(str)''
+'''syntax:''' ''newstr = ngx.escape_uri(str, type?)''
 
 '''context:''' ''init_by_lua*, init_worker_by_lua*, set_by_lua*, rewrite_by_lua*, access_by_lua*, content_by_lua*, header_filter_by_lua*, body_filter_by_lua*, log_by_lua*, ngx.timer.*, balancer_by_lua*, ssl_certificate_by_lua*, ssl_session_fetch_by_lua*, ssl_session_store_by_lua*''
 
 Escape <code>str</code> as a URI component.
 
+Since `v0.10.16rc6`, this function accepts an optional <code>type</code> argument.
+Note that this method accepts all [[#Nginx escape type constants|Nginx escape type constants]] as input.
```

Comment:
We can simply document the magical number 1, 2, and etc here, with more explanation on what these types really mean.